ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -250,9 +248,11 @@ StackLayout {
             active: {
                 if (root.communitySettingsDisabled)
                     return false
+                if (root.isMobile)
+                    return false
                 if (root.isAdmin || root.isTokenMaster)
                     return root.isPendingOwnershipRequest // not allowed for admin or TM unless there's the pending request
-                return true
+                return root.isControlNode || root.isTokenDeployed
```

Comment:
Here we show/hide the footer as previously done with `isPromoteSelfToControlNodeEnabled: root.isControlNode || root.isTokenDeployed`